### DISCUSSION TITLE
v1.5.0 Draft release

### DISCUSSION THREAD

**Discussion by Markemp** [Markemp/Cryengine-Converter#149] [Announcements]
- Fix for Ivo armatures, as well as general cleanup.
- Clarification on the Half data structure (`CryHalf`, `DymekHalf` and traditional IEEE `Half`).
- Renamed `Utils` to `Utilities`.  Why?  Seemed like a good idea at the time.

![image](https://user-images.githubusercontent.com/12057128/210287866-60da00f5-7ff5-4eaf-827e-b38616f612fe.png)

<hr /><em>This discussion was created from the release <a href='https://github.com/Markemp/Cryengine-Converter/releases/tag/v1.5.0'>v1.5.0 Draft release</a>.</em>

**Comment by Markemp** [comment]
Released!

**Reply by PotatoHate** [reply]
Hi, can you update Readme with these new changes

**Reply by Markemp** [reply]
Good call.  Updated.

**Comment by Baconator650** [comment]
Placed the new version in the StarFab utility, works pretty great with the landing gear, but noticed that the material naming doesn't include the mtl file anymore. Did the command change for this?

**Comment by odzta** [comment]
This work is freakin awesome! Thanks for creating this and making it available for general use! What a legend.

**Reply by Markemp** [reply]
Thanks, I really appreciate it!  It's still being actively worked on, with some really passionate (and far smarter than me) people helping out.  Glad you like it, and please submit feedback so we know how people are using it and where to focus our efforts on improving it.
